UnityProject/Assets/Scripts/Clothing/Eyewear/HUDItemBase.cs

```diff
@@ -0,0 +1,102 @@
+using System.Collections;
+using System.Collections.Generic;
+using Mirror;
+using UnityEngine;
+
+public class HUDItemBase : NetworkBehaviour,  IItemInOutMovedPlayer, IClientSynchronisedEffect
+{
+	protected Pickupable pickupable;
+
+	private void Awake()
+	{
+		pickupable =  GetComponent<Pickupable>();
+	}
+
+	#region InventoryMove
+
+	public RegisterPlayer CurrentlyOn { get; set; }
+	bool IItemInOutMovedPlayer.PreviousSetValid { get; set; }
+
+	void IItemInOutMovedPlayer.ChangingPlayer(RegisterPlayer HideForPlayer, RegisterPlayer ShowForPlayer)
```

Comment:
`private void`

Reply:
also dont put functions in the middle of variables

Reply:
Is automatically private, you can't access it when accessing the HUDItemBase class

Reply:
though if you cast to IItemInOutMovedPlayer it is then visible